apps/condo/pages/tls.tsx

```diff
@@ -0,0 +1,396 @@
+import enIos from '@condo/lang/en/pages/tls/ios.md'
+import enLinux from '@condo/lang/en/pages/tls/linux.md'
+import enMacos from '@condo/lang/en/pages/tls/macos.md'
+import enWindows from '@condo/lang/en/pages/tls/windows.md'
+import ruAndroid from '@condo/lang/ru/pages/tls/android.md'
+import ruIos from '@condo/lang/ru/pages/tls/ios.md'
+import ruLinux from '@condo/lang/ru/pages/tls/linux.md'
+import ruMacos from '@condo/lang/ru/pages/tls/macos.md'
+import ruWindows from '@condo/lang/ru/pages/tls/windows.md'
+import styled from '@emotion/styled'
+import { Anchor, Col, Collapse, Image, Row } from 'antd'
+import get from 'lodash/get'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import React, { CSSProperties, useCallback } from 'react'
+
+import { QuestionCircle } from '@open-condo/icons'
+import { useAuth } from '@open-condo/next/auth'
+import { useIntl } from '@open-condo/next/intl'
+import { Markdown, Typography } from '@open-condo/ui'
+import { Button, Card, Space } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+import EmptyLayout from '@condo/domains/common/components/containers/EmptyLayout'
+import { useLayoutContext } from '@condo/domains/common/components/LayoutContext'
+import { LoginWithSBBOLButton } from '@condo/domains/common/components/LoginWithSBBOLButton'
+import { Logo } from '@condo/domains/common/components/Logo'
+import { Poster } from '@condo/domains/common/components/Poster'
+import { PosterWrapper } from '@condo/domains/user/components/containers/styles'
+
+const LOGO_HEADER_STYLES = { width: '100%', justifyContent: 'space-between' }
+const HEADER_LOGO_STYLE: React.CSSProperties = { cursor: 'pointer' }
+
+const guidesContent = {
+    ru: {
+        windows: ruWindows,
+        macos: ruMacos,
+        linux: ruLinux,
+        android: ruAndroid,
+        ios: ruIos,
+    },
+    en: {
+        windows: enWindows,
+        macos: enMacos,
+        linux: enLinux,
+        android: enAndroid,
+        ios: enIos,
+    },
+}
+
+export const Header: React.FC = (): JSX.Element => {
+    const intl = useIntl()
+    const BackMessage = intl.formatMessage( { id: 'pages.condo.tls.back' })
+
+    const { push } = useRouter()
+    const { isAuthenticated } = useAuth()
+
+    const handleLogoClick = useCallback(() => {
+        if (isAuthenticated) {
+            push('/')
+        } else {
+            push('/auth/signin')
+        }
+    }, [isAuthenticated, push])
+
+    return (
+        <Row style={LOGO_HEADER_STYLES}>
+            <Col style={HEADER_LOGO_STYLE}>
+                <Logo onClick={handleLogoClick}/>
+            </Col>
+            <Col>
+                <Typography.Text>
+                    <Link href='/auth/signin'>
+                        <Typography.Link>{BackMessage}</Typography.Link>
+                    </Link>
+                </Typography.Text>
+            </Col>
+        </Row>
+    )
+}
+
+const CenteredText = styled.div`
```

Comment:
You don't need to use styled components just for centering text content. Rather you can do it with style constant or components, which already had this feature

Reply:
Implemented in https://github.com/open-condo-software/condo/pull/3537/commits/b132cfbae00abdbbbdfb4bc930f9bcb86d80b13c

Reply:
Discussed with @SavelevMatthew, no customization of open-condo/ui should be available.
Used React.CSSProperties constant instead